tensorboard/plugins/graph/tf_graph_common/parser.ts

```diff
@@ -38,15 +38,14 @@ function parseValue(value: string): string|number|boolean {
  * Fetches a text file and returns a promise of the result.
  */
 export function fetchPbTxt(filepath: string): Promise<ArrayBuffer> {
-  return new Promise<ArrayBuffer>(function(resolve, reject) {
-    const request = new XMLHttpRequest();
-    request.open('GET', filepath);
-    request.responseType = 'arraybuffer';
-
-    request.onerror = () => reject(request.status);
-    request.onload = () => resolve(request.response);
-
-    request.send(null);
+  return new Promise((resolve, reject) => {
+    fetch(filepath).then(res => {
```

Comment:
style:

> Tip: It is a good practice to use parentheses even for single-argument
> arrows, since the code may still parse reasonably (but incorrectly) if
> the parentheses are forgotten when an additional argument is added.

<https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions>